docs/usage/shoot_firewall.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Shoot Firewall Configuration
+weight: 15
+---
+
+# Endpoints and Ports of a Shoot Control-Plane
+
+With the [reversed VPN](../operations/reversed-vpn-tunnel.md) tunnel, there are no endpoints with open ports in the shoot cluster required by Gardener.
+In order to allow communication to the shoots control-plane in the seed cluster, there are endpoints shared by multiple shoots of a seed cluster.
+Depending on the configured zones or [exposure classes](./nessesary_considerations/exposureclasses.md), there are different endpoints in a seed cluster. The IP address(es) can be determined by a DNS query for the API Server URL.
+The main entry-point into the seed cluster is the load balancer of the Istio ingress-gateway service. Depending on the infrastructure provider, there can be one IP address per zone.
```

Comment:
> Depending on the infrastructure provider, there can be one IP address per zone.

Is it really depending on the infrastructure provider or how the istio is configured in the Seed spec?

Reply:
This was copy-pasted from `Endpoints and Ports of a Shoot Control-Plane`